src/prefect/engine/executors/local.py

```diff
@@ -9,13 +9,16 @@ class LocalExecutor(Executor):
     the main thread.  To be used mainly for debugging purposes.
     """
 
-    def submit(self, fn: Callable, *args: Any, **kwargs: Any) -> Any:
+    def submit(
+        self, fn: Callable, *args: Any, executor_kwargs: dict = None, **kwargs: Any
+    ) -> Any:
         """
         Submit a function to the executor for execution. Returns the result of the computation.
 
         Args:
             - fn (Callable): function that is being submitted for execution
             - *args (Any): arguments to be passed to `fn`
+            - executor_kwargs (dict, optional): an optional dictionary of executor-specific keyword arguments
```

Comment:
Since we pass the same kwargs everywhere, they're not really executor specific. Perhaps either rename to `extra_context` (the context for this task is the current context in scope, plus the `extra_context` mapping), or explicitly codify the kwargs forwarded as `task_name` and `task_tags`?

Reply:
Yea I'm definitely game to change the name.  The reason I chose `executor_kwargs` is that they represent kwargs that the executor may / may not utilize, but this is a low-risk decision either way so I'll update to `extra_context` 👍

Reply:
Updated

Reply:
Yeah, to me `executor_kwargs` implies kwargs that are directly forwarded to the backing executor in some way, which isn't true here. Update looks good.